modules/network-peering/variables.tf

```diff
@@ -29,3 +29,15 @@ variable "peer_network" {
   description = "Resource link of the peer network."
   type        = string
 }
+
+variable "exchange_custom_routes_to_local" {
```

Comment:
Can we find better variables and support more granularity for custom routes? Eg I might want to export from local and import in peers, but not vice-versa (legitimate use case).

I would use something like `local_export_routes`, `local_import_routes`, `peer_export_routes`, `peer_import_routes`.

Reply:
Hmmmm after chatting, this might work. I would still change names though: `local_export_routes` (implicitly imports in peer), and `peer_export_routes` (implicitly imports in local).